docs/tutorial/using-a-theme.md

```diff
@@ -58,31 +62,79 @@ module.exports = {
     ],
   },
   {/* highlight-end */}
+  plugins: [
+    {
+      resolve: "gatsby-theme-blog",
+      options: {},
+    },
+  ],
 }
 ```
 
+## Add some content
+
+Before you can see anything, you'll want to add some content so there is something to show.
+
+By default, the posts are expected in the `/content/posts` directory, so create those folders and add a `my-post.md` file. Your file structure should look something like this.
+
+```text
+my-blog
+├── content
+│   └── posts
+│       └── my-post.md
+├── src
+├── gatsby-config.js
+└── package.json
+```
+
+Inside that Markdown file, add content. The top section is called [frontmatter](/docs/mdx/writing-pages/#using-frontmatter-in-mdx) and `title` and `date` are required fields.
+
+````markdown:title=/content/posts/my-post.md
+---
+title: My Post
+date: 2020-04-15
+---
+
+Let's write a post!
+
+```javascript
+const test = "this is a theme"
+```
+````
+
+## Test run your site
+
+To make sure everything is working, run your site. This command should be run in your terminal in your project's root directory.
+
+```shell
+gatsby develop
+```
+
+Navigate to `http://localhost:8000` to see the landing page of your site.
```

Comment:
I'm assuming based on the tutorial that I should be seeing something at whatever path I specified. Is that correct? Right now, I'm getting a 404.